ProcessLib/GroundwaterFlowFEM.h

```diff
@@ -109,8 +114,17 @@ class LocalAssemblerData : public LocalAssemblerDataInterface<GlobalMatrix, Glob
     void addToGlobal(GlobalMatrix& A, GlobalVector& rhs,
             AssemblerLib::LocalToGlobalIndexMap::RowColumnIndices const& indices) const
     {
-        A.add(indices, *_localA);
-        rhs.add(indices.rows, *_localRhs);
+#ifdef USE_PETSC
```

Comment:
This part must go into the `addToGlobal` function. It will not be possible to maintain the PETSc support for the future process implementations.